chapters/VK_EXT_debug_utils.txt

```diff
@@ -647,8 +654,15 @@ The parameters are passed on to the callback in addition to the
 pname:pUserData value that was defined at the time the messenger was
 registered.
 
+.Valid Usage
+****
+  * For each object in pname:pCallbackData\->pname:pObjects,
+    pname:objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN.
```

Comment:
maybe
> pname:objectType member of each element of pname:pCallbackData\\->pname:pObjects must: not be ename:VK_OBJECT_TYPE_UNKNOWN

?
Also, no period for consistency.

Reply:
:+1: